test/algodclientv2_test.go

```diff
@@ -362,3 +363,8 @@ func parsedDryrunResponseShouldHave(key string, action int) error {
 	}
 	return nil
 }
+
+func mockHttpResponsesInLoadedFrom(jsonfiles, directory string) error {
+	fullPath := path.Join("./features/unit/", directory)
```

Comment:
Could you change this to `features/resources`? The files in `unit` are also in `resources`, I'd like to delete `unit`. For example:
```
$ find . |grep pendingTransactionsByAddr.base64
./test/features/unit/v2algodclient_responsejsons/pendingTransactionsByAddr.base64
./test/features/resources/v2algodclient_responsejsons/pendingTransactionsByAddr.base64
```

Reply:
done. I guess I did not completely understand you before. 
Thanks.